dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtility.java

```diff
@@ -261,7 +267,9 @@ private static MavenProject createMavenProject(Path pomFile, RepositorySystemSes
         logger.severe("Duplicate dependency " + dependency);
       }
     }
-    return managedDependencies;
+    
+    Bom bom = new Bom(Artifacts.toCoordinates(artifact), ImmutableList.copyOf(managedDependencies));
```

Comment:
If second argument is `List` then copyOf is not needed.

Reply:
ack